bot/src/main/java/org/openjdk/skara/bot/BotRunner.java

```diff
@@ -231,19 +231,24 @@ private void runMeasured() {
                         log.log(Level.SEVERE, "Exception during item execution (" + item + "): " + e.getMessage(), e);
                     }
                     item.handleRuntimeException(e);
+                } catch (Error e) {
+                    EXCEPTIONS_COUNTER.labels(item.botName(), item.workItemName(), e.getClass().getName()).inc();
+                    log.log(Level.SEVERE, "Error thrown during item execution: " + e.getMessage(), e);
```

Comment:
Maybe we could also log `item` in the message.

Reply:
Right yes, that's a good idea.